docs/api/read-model/store.md

```diff
@@ -143,10 +152,10 @@ Returns the number of items that meet the specified condition.
 
 #### Arguments
 
-| Argument Name   | Description                                   |
-| --------------- | --------------------------------------------- |
-| tableName       | A table name.                                 |
-| searchCondition | An object that defines a search expression.   |
+| Argument Name   | Description                                 |
+| --------------- | ------------------------------------------- |
+| tableName       | A table name.                               |
+| searchCondition | An object that defines a search expression. |
 
 #### Example
 
```

Comment:
Please update the following lines below:

Line 104:
// Return all but the specified fields

Can we update this line as follows?

// Return all fields except for the specified fields

Line 248:
Search expression use operators to compare values and group expression clauses.

Please change "Search expression use" to "Search expressions use".

Line 260:
Matches values that are greater then the specified value.

Please change "then" to "than".

Line 261:
Matches values that are greater then or equal to the specified value.

Please change "then" to "than".

Reply:
Thanks.